src/commands/list.rs

```diff
@@ -10,6 +10,10 @@ pub fn list_files(args: &[String])
     let image_fn = args[0].clone();
     let image = fat::Image::from(image_fn)?;
 
+    let params = image.bios_parameter();
+    // Just dump for testing. Remove when tried of me :-)
+    println!("{:?}", params);
```

Comment:
Works for me. I was mostly dumping to show functionality since it isn't used for anything yet :-)  Will fix.